tests/test_cal_announce.py

```diff
@@ -7,10 +10,25 @@ def test_get_cal():
     assert res["statusCode"] == 200
 
 
-def test_missing_tok():
-    real = cal_announce.token_path
-    cal_announce.token_path = "./bogus.pickle"
-    
+@patch.object(GOOGLE_CAL, 'CAL_API_KEY', '')
```

Comment:
owo, is a this dependency injection

Reply:
Yup; just doing it one line now instead of 3 (like in some of the other tests). I will make a separate PR to cleanup some of the tests we have